lldb/test/API/lang/swift/metadata_cache/TestSwiftMetadataCache.py

```diff
@@ -0,0 +1,121 @@
+# TestSwiftMetadataCache.py
+#
+# This source file is part of the Swift.org open source project
```

Comment:
Ever since swift and llvm share the same license we no longer need to put these headers into every file.